veracruz-server/src/cli.rs

```diff
@@ -47,27 +46,15 @@ fn main() {
     };
     info!("Loaded policy {}", policy.policy_hash().unwrap_or("???"));
 
-    // create Actix runtime
-    let sys = actix_rt::System::new();
-
     // create Veracruz Server instance
-    let veracruz_server = match veracruz_server::server::server(&policy_json) {
-        Ok(veracruz_server) => veracruz_server,
-        Err(err) => {
-            eprintln!("{}", err);
-            process::exit(1);
-        }
-    };
+    veracruz_server::server::server(&policy_json).unwrap();
 
     println!(
         "Veracruz Server running on {}",
         policy.veracruz_server_url()
     );
-    match sys.block_on(veracruz_server) {
-        Ok(_) => {}
-        Err(err) => {
-            eprintln!("{}", err);
-            process::exit(1);
-        }
+
+    loop {
+        std::thread::sleep(std::time::Duration::MAX);
```

Comment:
Is this necessary or desirable? I would think if the server returns, we the server has exited. It will `panic` on any error, and not `panic` on... maybe never. In any case, if the server does exit, I would like my program to exit and not to just hang.

Reply:
I've answered this on #547.